spec/post_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  # tests go here
+  subject { Post.new(author: User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.'), title: 'Hello', text: 'This is for testing') }
+
+  before { subject.save }
+
+  it 'expects post title to be invalid' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'User post counter to increment' do
+    subject.send(:update_posts_counter)
+    expect(subject.author.posts_counter).to be(1)
+  end
+
+  it 'Likes Counter attribute should be greater or equal to zero' do
+    subject.likes_counter = -1
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Post spec is missing most of the tests for the post model, Please make sure to add tests for all methods and validations described in the post model.

Below are the validations and methods
![image](https://github.com/Caren-Koroeny/caren_blog-app/assets/83803414/92739d08-04d4-414d-bcb6-88c748ca213d)